flag_ext.go

```diff
@@ -61,3 +158,61 @@ func (e *extFlag) GetDefaultText() string {
 func (e *extFlag) GetEnvVars() []string {
 	return nil
 }
+
+// RunAction executes flag action if set
+func (e *extFlag) RunAction(ctx context.Context, cmd *Command) error {
```

Comment:
you dont need to define this method. interface casting will check if Flag is an ActionableFlag and call accordingly